docs/application/dotnet/guides/multimedia/camera.md

```diff
@@ -146,19 +146,23 @@ To configure the camera:
 
     The [Tizen.Multimedia.CameraPixelFormat](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.CameraPixelFormat.html) enumeration defines the available capture formats.
 
-5.  Register event handlers for managing various events of the `Tizen.Multimedia.Camera` class, related to the camera preview, auto-focus, and capturing:
+5.  Register event handlers for managing various events of the `Tizen.Multimedia.Camera` class, related to the camera preview, autofocus, and capturing:
     -   <a name="callbacks_preview"></a>To handle the camera preview, register an event handler for the `Preview` event. The event handler is invoked once per frame during a preview.
```

Comment:
Kindly remove the <a> tag because in MD files avoid using <a> tags and here it is not hyperlinking any text or resource.

Reply:
Other document I've seen used tags to link internally.
How can I use MD style for internal link?
Is below example right?

```
If you want to configure camera, please refer [Configure camera](#anchor-conf1) section

...

##Anchor conf1
Configure camera
1. step 1
2. step 2
```

Reply:
And why do we have to avoid tags or anchor tags in MD file? It's working well until now.

Reply:
@hsgwon: Above example is correct way of hyperlinking in MD files. The other files where anchor tag is given for hyperlinking within pages is because those are hyperlinking sentences or paragraphs. As MD allows only to hyperlink titles/sub-titles only within a page.

Reply:
And How can I archive below example?

```
You can set event handler to control the some event. 
( => I want to link below 5. To handle~~ statement into 'set event handler' statement here, and I don't want to make '5. To handle~~' statement bigger and bold word)

...

5. To handle the camera preview
```

Reply:
I read your comment just now, and the usage of my just previous question is not available, right?